ARC17_OBJECT_NAMES : [
Toilet_Brush,
Avery_Binder,
Balloons,
Band_Aid_Tape,
Bath_Sponge,
Black_Fashion_Gloves,
Burts_Bees_Baby_Wipes,
Colgate_Toothbrush_4PK,
Composition_Book,
Crayons,
Duct_Tape,
Epsom_Salts,
Expo_Eraser,
Fiskars_Scissors,
Flashlight,
Glue_Sticks,
Hand_Weight,
Hanes_Socks,
Hinged_Ruled_Index_Cards,
Ice_Cube_Tray,
Irish_Spring_Soap,
Laugh_Out_Loud_Jokes,
Marbles,
Measuring_Spoons,
Mesh_Cup,
Mouse_Traps,
Pie_Plates,
Plastic_Wine_Glass,
Poland_Spring_Water,
Reynolds_Wrap,
Robots_DVD,
Robots_Everywhere,
Scotch_Sponges,
Speed_Stick,
White_Facecloth,
Table_Cloth,
Tennis_Ball_Container,
Ticonderoga_Pencils,
Tissue_Box,
Windex,
a,b,c,d,e,f,g,h,i,j
]
